### PR TITLE
Adapt roles

### PR DESCRIPTION
**Pull request by jreidinger**

### CODE REVIEW

File: control/control.leanos.xml [hunk 1]
```diff
@@ -139,7 +139,8 @@ textdomain="control"
             </one_supported_desktop>
         </supported_desktops>
 
-        <default_desktop>gnome</default_desktop>
+        # use default desktop order based on installed packages
```
Comment (lslezak) [inline control/control.leanos.xml]: ⚠️ Parse Error.

Something like `no default desktop, can be redefined by an add-on` sounds better...

**Review by lslezak — CHANGES_REQUESTED** [review]
Comment?

File: control/control.leanos.xml (hunk 1 above)
Comment (jreidinger) [reply control/control.leanos.xml]: actually not. This is for https://trello.com/c/NmLDu0Gz/1730-bring-back-icewm

`It is do not define default desktop in control file and use instead update-alternatives approach` . Is it better?

Comment (lslezak) [reply control/control.leanos.xml]: OK, even better! 👍

Comment (lslezak) [reply control/control.leanos.xml]: After reading again I'd suggest just a better wording: `Do not define the default desktop in control file, use the update-alternatives approach instead`

**Review by lslezak — APPROVED** [review]
LGTM

**Pull request merged**